### PR TITLE
Address Installation Dependency Gaps

### PR DESCRIPTION
**Pull request by d33bs**
# Description

When installing Pycytominer using pip from Github hashes there is a dependency gap with `pyarrow` due to mismatches between `requirements.txt` and `setup.py`. This currently causes new functionality to be otherwise unavailable until the dependency is resolved manually.

Changes in this PR update `setup.py` to reference `requirements.txt` when installing Pycytominer. I've also added capitalization for globals to help make these more visible from a style perspective.

Many thanks go to @axiomcura for help in highlighting this issue!

## What is the nature of your change?

- [x] Bug fix (fixes an issue).
- [ ] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,23 +2,29 @@
 from setuptools import setup
 from setuptools import find_packages
 
+LONG_DESCRIPTION = ""
```
Comment (gwaybio) [inline setup.py:5]: why do you need to set this empty string? Won't line 7 stand alone?

File: setup.py [hunk 2]
```diff
@@ -2,23 +2,29 @@
...
-about = {}
+ABOUT = {}
 with open(pathlib.Path("pycytominer/__about__.py")) as fp:
-    exec(fp.read(), about)
-print(about)
+    exec(fp.read(), ABOUT)
+
+# pull requirements for install_requires
+REQUIRED_PKGS = []
+with open('requirements.txt') as f:
```
Comment (gwaybio) [inline setup.py:15]: Please make sure to `black` this file too! :) (I notice this line b/c of single quotes)

**Review by gwaybio — CHANGES_REQUESTED** [review]
One comment and one requested change (run `black` please)

File: setup.py (hunk 1 above)
Comment (d33bs) [reply setup.py:5]: Great point, I was following form with the other `open` shortly after this one. I'll remove this and the other similar instantiation for the REQUIRED_PKGS to simplify.

File: setup.py (hunk 2 above)
Comment (d33bs) [reply setup.py:15]: Will do, apologies about this.

Comment (gwaybio) [reply setup.py]: No worries!

**Review by gwaybio — APPROVED** [review]

**Comment by gwaybio** [comment]
I will merge once @falquaddoomi gives his 👍 - thanks @d33bs !

**Review by falquaddoomi — APPROVED** [review]

**Pull request merged**